### DISCUSSION TITLE
Issue with AWS linux2

### DISCUSSION THREAD

**Discussion by mrodclabs** [Xpra-org/xpra#3736] [General]
hi all,

I'm having an issues running this on amazon linux2 

This is the current version: 
```
NAME="Amazon Linux"
VERSION="2"
ID="amzn"
ID_LIKE="centos rhel fedora"
VERSION_ID="2"
PRETTY_NAME="Amazon Linux 2"
ANSI_COLOR="0;33"
CPE_NAME="cpe:2.3:o:amazon:amazon_linux:2"
HOME_URL="https://amazonlinux.com/"
```
the error i'm getting is: 
```
https://xpra.org/dists/Fedora/2/x86_64/repodata/repomd.xml: [Errno 14] HTTPS Error 404 - Not Found
```

I know the docs don't mentioned specifically linux2, only redhat and centos but as far as i know this should work

I use the command listed to add the repo.
```
wget -O /etc/yum.repos.d/xpra.repo https://xpra.org/repos/CentOS/xpra.repo
```
Any suggestions to get this working on linux2 ?

**Comment by totaam** [comment]
Your repository URL is wrong.
There is no `Fedora/2` in the repositories and whatever `Amazon Linux 2` is, it doesn't look like a supported OS.

> I use the command listed to add the repo.

No, you did not.
This command would have added the `CentOS` repo, not the `Fedora` one.

> Any suggestions to get this working on linux2 ?

AFAIK, `linux2` is not a distribution, it is just a codename.
Change your repository file to point to the `RHEL` repository matching your OS version.
ie: `RedHat/8` or `RedHat/9`.

**Reply by mrodclabs** [reply]
Thanks, i found it: redhat7 in case anyone wonders in the future.

**Reply by totaam** [reply]
I very much doubt that `redhat7` is the correct value as there is no such directory in the repositories - I assume that you've used `RedHat/7` instead.

**Comment by mrodclabs** [comment]
```
[xpra]
name=Xpra $basearch
enabled=1
metadata_expire=1d
gpgcheck=1
gpgkey=https://xpra.org/gpg.asc
       https://xpra.org/gpg-2022.asc
baseurl=https://xpra.org/dists/RedHat/7/$basearch/
```

This is how i made it work

**Comment by totaam** [comment]
See also: https://github.com/Xpra-org/xpra/issues/3863
